brave/brave-ios
# Fix #916: Add pull to refresh

## Summary of Changes


This pull request fixes #916

## Submitter Checklist:


## Test Plan:

- Verify you can pull to refresh on normal websites
- Verify you cannot pull to refresh on local pages/error pages

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
